source/content/guides/drupal-9/drupal-9-hosted-pre112021/07-switch-drupal.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Migrate a Site That Was Created Before November 2011 to Drupal 9
+subtitle: Switch to Drupal with Composer Upstream
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-pre112021/switch-drupal
+anchorid: switch-drupal
+editpath: drupal-9/drupal-9-hosted-pre112021/07-switch-drupal.md
+---
+
+This page provides steps to switch your site from Drupal to [Drupal with Composer Upstream](/guides/integrated-composer#get-started-with-integrated-composer).
+
+1. Change the upstream that your site is tracking with the following command:
+
+    ```bash{promptUser:user}
+    terminus site:upstream:set $SITE drupal-recommended
+    ```
+
+1. Follow the `drupal-recommended` upstream to keep your site current with general configuration changes recommended by Pantheon.
```

Comment:
Honestly, I'm not sure, it was carried over from the previous documentation.  It previously said "The `drupal-recommended` Integrated Composer upstream works with Drupal 8, and following the `drupal-recommended` upstream will help keep your site up to date with any general configuration changes recommended by Pantheon.", but Kevin edited it to what it is now.

Reply:
@kporras07 should we add steps from another doc in the [migrate guides](https://pr-7235--pantheon-docs.my.pantheonfrontend.website/docs/drupal-9-migration) to this section? seems like this step wants us to do a thing that might be documented on another page

Reply:
@EdwardAngert not really. This is the current source document: https://pantheon.io/docs/guides/switch-drupal-recommended-upstream#switch-to-drupal-with-composer-upstream and it's just referring to the fact that staying in that upstream will keep updates coming to your dashboard

Hope it helps

Reply:
thanks @wordsmither and @kporras07 

I modified this section to

```markdown
This page provides steps to switch your site from Drupal to `drupal-recommended`, the [Drupal with Composer Upstream](/guides/integrated-composer#get-started-with-integrated-composer) upstream that keeps your site current with general configuration changes recommended by Pantheon.

Use Terminus to change the upstream that your site is tracking:

\```bash{promptUser:user}
terminus site:upstream:set $SITE drupal-recommended
\```
```

if that seems to describe it correctly, I'll 👍  and we can merge

Reply:
This looks good.

Since this is a loooot of changes, I'd like to do another round of technical review.

Also, since this work started, the recommended upstream changed, so I'm sure there are some references to the old upstream in this branch.